price_feeder/tasks.py

```diff
@@ -411,6 +462,9 @@ def task_price_feed(self, task=None, global_manager=None):
         # price variation accepted
         price_variation_write_blockchain = self.options['networks'][self.config_network]['price_variation_write_blockchain']
 
+        # re post price variation accepted
+        price_variation_re_write_blockchain = self.options['networks'][self.config_network]['price_variation_re_write_blockchain']
```

Comment:
can we plz abstract `self.options['networks'][self.config_network]`, hurts my eyes. :P